homeassistant/components/braviatv/coordinator.py

```diff
@@ -122,6 +122,9 @@ async def _async_update_data(self) -> None:
                 await self.async_update_sources()
             await self.async_update_volume()
             await self.async_update_playing()
+        except BraviaTVNotFound:
+            self.connected = False
```

Comment:
I would prefer if a timed counter was here which actually starts bubbling up the 404 exceptions after e.g. 5 minutes from the first 404.